nix/project.nix

```diff
@@ -0,0 +1,160 @@
+# editorconfig-checker-disable-file 
+
+{ repoRoot, inputs, lib, ... }:
+
+let
+  cabalProjectArgs = { config, pkgs, ... }: {
+
+    name = "plutus";
+
+    compiler-nix-name = lib.mkDefault "ghc92";
+
+    src = ../.;
+
+    # We would expect R to be pulled in automatically as it's a dependency of
+    # plutus-core, but it appears it is not, so we need to be explicit about
+    # the dependency on R here. Adding it as a buildInput will ensure it's
+    # added to the pkg-config env var.
+    shell = {
+      withHoogle = false;
+      buildInputs = [ pkgs.R ];
+    };
+
+    flake.variants = {
+      ghc92 = { }; # Alias for the default project
+      ghc92-profiled.modules = [{ enableProfiling = true; }];
```

Comment:
Is this enough? I have a vague feeling that we also need `enableLibraryProfiling`?

Reply:
Good catch, thanks.